src/timing.jl

```diff
@@ -9,9 +9,8 @@ export every, fpswhen, fps, timestamp
 function every(delta::Float64)
     i = Input(time())
     update(timer) = push!(i, time())
-    t = Timer(update)
-    start_timer(t, delta, delta)
-    return lift(x->x, i, typ=Float64) # prevent push!
+    t = Timer(update, delta, delta)
```

Comment:
If you add a dependency to Compat 0.4.6 in `REQUIRE`, this should work on v0.3 as well.